deployments/static/extensions-v1beta1-nvidia-device-plugin.yml

```diff
@@ -35,7 +35,7 @@ spec:
       # See https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
       priorityClassName: "system-node-critical"
       containers:
-      - image: nvcr.io/nvidia/k8s-device-plugin:v0.14.4
+      - image: nvcr.io/nvidia/k8s-device-plugin:v0.15.0-rc.1
```

Comment:
As part of "Remove `legacyDaemonsetAPI` config option. This was only required for k8s versions < 1.16." I would have just dropped this whole file.

Reply:
Which file?

Reply:
this whole file `deployments/static/extensions-v1beta1-nvidia-device-plugin.yml`

Reply:
OK. Will do as a cleanup. Thanks.